docker-compose.yml

```diff
@@ -44,8 +44,20 @@ services:
     environment: 
       MPS_USERNAME: ${MPS_USER}
       MPS_PASS: ${MPS_PASSWORD}
+      MPS_INSTANCE_NAME: '{{.Task.Name}}'
     volumes:
       - private-volume:/mps-microservice/private
+  mpslookup:
+    image: mpslookup:latest
+    environment:
+      MPS_CONNECTION_STRING: postgresql://postgresadmin:admin123@db:5432/mpsdb?sslmode=disable
+    ports:
+      - "8003:8003"
```

Comment:
I don't think we need to keep this exposed do we?

Reply:
done